src/workloads/query/CsiFragmentedInsertsNested.yml

```diff
@@ -0,0 +1,167 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares performance of inserts into a collection with only the default _id index
+  and in presence of a full columnstore index. We are not comparing to wildcard index because the
+  nested data makes creating of a wildcard index too slow. Before changing any of the parameters in
+  this workload please make sure the results can be correlated with 'CsiFragmentedInsertsFlat.yml'.
+  As the approach in this workload is the same as in 'CsiFragmentedInsertsFlat.yml' with the exception
+  of data used by the loader (and not comparing to the wildcard index), comments are intentionally
+  omitted, please refer to the "flat" workload for the details.
+
+Keywords:
+- columnstore
+- insert
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone-all-feature-flags
+    branch_name:
+      $neq:
+      - v4.0
+      - v4.2
+      - v4.4
+      - v5.0
+      - v6.0
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: 600_000  # = 10 min
+      connectTimeoutMS: 600_000
+
+GlobalDefaults:
+  MaxPhases: &maxPhases 5
+  Database: &db csiFragmentedInsertsNested
+  Collection: &coll Collection0
+  DocumentCount: &docCount 1e6
+  SchemaWidth: &schemaWidth 10000
+  ObjectWidth: &objectWidth 5  # with two nested paths, get 10 paths_per_object
+  SampleSize: &sampleSize 10
+
+  Document: &document
```

Comment:
You can (don't have to) use LoadConfig for the common features between these two files. https://github.com/mongodb/genny/blob/3681cd089bb340eef3ed4a3643074d6a4171bc4a/docs/using.md#orga6d35c7

For example, I wouldn't be surprised if your two InsertFromSample templates are identical. You could write it once and load it into each file.

Reply:
I'd like to keep the file for Flat workload as readable as possible on its own. Would it be OK, rather than loading some common file into both workload, to load into Nested from Flat?

Reply:
Yeah, that should be fine.

Reply:
Also, only if it makes sense to you.

Reply:
Actually, the two workloads differ only in three parameters: object width, db name and the document generator. Is it possible to have a single file that describes the test but takes these three parameters to create two separate tasks?

Reply:
I'm not sure what the best path is. I could get it to two files, with one of the files being very small. I've asked in slack and tagged you on the thread.

Reply:
Genny has no option for a single YAML file to produce multiple tasks.

This came up in a couple other conversations recently, so I think it's worth considering as a new syntax thing.

I think this fits with AutoRun pretty nicely:

```yaml
Actors:
- Type: {^Parameter: {Name: ActorType}}
AutoRun:
- When:
    mongodb_setup:
      $eq: replica
  ThenRun:  # Doesn't work, just a sketch.
  - DSIParameters:
      mongodb_setup: replica-delay-mixed
    GennyParameters:
      ActorType: Delayed
  - DSIParameters:
      mongodb_setup: replica-delay-mixed
    GennyParameters:
      ActorType: NotDelayed
```

This would be surprisingly low effort to add, but tbh I don't super love the syntax. And Genny's yaml syntax is already reaching critical mass of understandability, so I want to tread carefully here. (E.g. I completely forgot that `ActorTemplates` was a thing.)

ANYWAY. The end state here is subjective unless we invest in more syntax.

My $0.02 is to opt for a single parameterized workload yaml file that has these three parameters defined. Then two yaml files that just provide the parameters. Class and instance style kinda.

Reply:
Thanks! Would you mind pointing me to an existing example of such parameterized workloads (ideally, that are AutoRun)?

Reply:
I'm going to merge as is. I'll be working on similar workloads for update/delete soon and can refactor this one at the same time, after I learn more about using genny _well_ (it doesn't come naturally, btw :P)